cvise/passes/clangbinarysearch.py

```diff
@@ -2,10 +2,11 @@
 import os
 import re
 import shutil
+import subprocess
 import tempfile
 import time
 
-from cvise.passes.abstract import AbstractPass
+from cvise.passes.abstract import *
```

Comment:
Please changes this to:
```
from cvise.passes.abstract import AbstractPass, BinaryState, PassResult
```

so that LGTM is happy.

Reply:
Aha, makes sense. Done.